packages/router/README.md

```diff
@@ -43,9 +43,9 @@
     <span> | </span>
     <a href="https://github.com/DioxusLabs/example-projects"> Examples </a>
     <span> | </span>
-    <a href="https://dioxuslabs.com/guide"> Guide (0.1.8) </a>
+    <a href="https://dioxuslabs.com/router"> Guide (0.2.4) </a>
```

Comment:
perhaps we can just indicate "(Latest)" instead of having to update this with every release?